src/components/ContentEditor/WidgetEditor/WidgetEditor.tsx

```diff
@@ -64,20 +60,36 @@ export const WidgetEditor: React.FC<WidgetEditorProps> = ({ content, contentHash
 
         {editing && <>
             <div>
-                <FormControl className="content-editor-formcontrol">
-                    <InputLabel>Select a component type</InputLabel>
-                    <Select
-                        value={editedContent.type}
-                        onChange={(e) => {
-                            setEditedContentOnChange("type", e.target.value as string, editedContent, setEditedContent);
-                        }}>
-                        {Object.keys(WidgetTypes).map(widgetType => {
-                            return <MenuItem key={widgetType} value={widgetType}>
-                                {widgetType}
-                            </MenuItem>
-                        })}
-                    </Select>
-                </FormControl>
+                <div className="wiki-form">
```

Comment:
Maybe `widget-picker` instead of `wiki-form`?

Reply:
Switched className to widget-picker and updated WidgetEditor.css accordingly